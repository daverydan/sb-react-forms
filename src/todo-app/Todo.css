.Todo {
  margin: 15px 0;
}

.Todo button:hover {
  cursor: pointer;
}