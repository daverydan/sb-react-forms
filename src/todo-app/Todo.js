import React from "react";
import "./Todo.css";

const Todo = ({ todo, id, removeTodo }) => {
  return (
    <div className="Todo">
      {todo}{" "}
      <button onClick={removeTodo} data-testid={`todo-btn-${id}`}>
        x
      </button>
    </div>
  );
};

export default Todo;
