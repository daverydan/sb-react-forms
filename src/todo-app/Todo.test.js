import React from "react";
import { render } from "@testing-library/react";
import Todo from "./Todo";

it("renders w/out crashing", () => {
  render(<Todo />);
});

test("it matches snapshot", () => {
  const { asFragment } = render(<Todo />);
  expect(asFragment()).toMatchSnapshot();
});
