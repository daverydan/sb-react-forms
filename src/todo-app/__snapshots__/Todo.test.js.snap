// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Todo"
  >
     
    <button>
      x
    </button>
  </div>
</DocumentFragment>
`;
